client/blocks/comments/post-comment-list.jsx

```diff
@@ -159,10 +170,13 @@ class PostCommentList extends React.Component {
 			nextProps.requestPostComments( { siteId, postId, status } );
 		}
 
-		if ( this.shouldScrollToComment( nextProps ) ) {
-			// defer so that the above/below comments have time to render
-			delay( () => this.scrollToComment(), 50 );
-		}
+		// first defer is to give the startingCommentId time to render to the dom
+		// second defer is to give the above/below comments to render to the dom
+		delay( () => {
+			if ( this.shouldScrollToComment( nextProps ) ) {
```

Comment:
i know this is carryover, but there's a chance that nextProps here is no longer the correct props.... since delay is somewhat indeterminate as to when it runs, we could have moved on from this set of props entirely.

maybe we could make sure nextProps and `this.props` are `===` before doing the check? would that work...

not sure how much i care?